src/Stepper/StepperContext.jsx

```diff
@@ -51,7 +56,7 @@ export function StepperContextProvider({ children, activeKey }) {
         registerStep,
         steps,
         removeStep,
-        getIsComplete,
```

Comment:
Yes, this function was only internal, consumers couldn’t access it since we don’t export context components (also checked usage insights to make sure)